llvm/test/Transforms/LoopVectorize/pr87407.ll

```diff
@@ -0,0 +1,70 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --check-globals none --version 5
```

Comment:
might be worth changing the test name to something slightly more descriptive, e.g. `pr87407-trunc-with-intrinsic.ll` or something like that